src/__tests__/theme-provider.js

```diff
@@ -30,6 +30,57 @@ test('renders a component with theme', () => {
   ).toMatchSnapshot()
 })
 
+test('renders inverted theme', () => {
+  const Comp = glamorous.div(({theme: {fg, bg}}) => ({
+    backgroundColor: bg,
+    color: fg,
+  }))
+  const theme = {
+    fg: 'palevioletred',
+    bg: 'white',
+  }
+  const invertTheme = ({fg, bg}) => ({
+    fg: bg,
+    bg: fg,
```

Comment:
Idk why, but I love this test you made. Slightly more interesting than the typical unit! 😅